vue/sbc-common-components/src/components/SbcAuthenticationOptions.vue

```diff
@@ -1,42 +1,48 @@
 <template>
   <v-container class="view-container">
-    <div class="view-header flex-column mb-10">
+    <div class="view-header flex-column mb-9">
       <h1 class="view-header__title">Log in to BC Registries</h1>
-      <p class="mt-2 mb-3">Don't have a BC Registries account?
-        <a class="text-decoration-underline" @click="goToCreateAccount">Create an account</a>
+      <p class="mt-4 mb-0">
+        Don't have a BC Registries account? <a class="text-decoration-underline" @click="goToCreateAccount">Create an account</a>
       </p>
     </div>
     <v-row>
       <v-col
+        sm="12"
+        md="6"
         class="d-flex align-stretch"
-        sm="12" md="6"
         v-for="authOption in authOptions"
         :key="authOption.type"
       >
         <v-card
           flat
-          class="account-card text-center pa-7 d-flex flex-column"
+          outlined
+          hover
+          class="account-card text-center pa-10 elevation-2 d-flex"
+          @click="selectAuthType(authOption)"
```

Comment:
Is this click is needed here? as per the design, we have the click on the button in each card

Reply:
I've update this to more closely match the patterns we've established elsewhere in the application (full-clickable cards)